Add stack direction for Blackfin

The stack grows downward on Blackfin, see
https://sourceware.org/binutils/docs/as/Blackfin-Syntax.html: "The
stack grows by decrementing the stack pointer.".

Signed-off-by: Thomas Petazzoni <thomas.petazzoni@free-electrons.com>
Index: b/m4/stack-direction.m4
===================================================================
--- a/m4/stack-direction.m4
+++ b/m4/stack-direction.m4
@@ -18,6 +18,7 @@
       arc | \
       arm* | strongarm* | xscale* | \
       avr | avr32 | \
+      bfin | \
       c1 | c2 | c32 | c34 | c38 | \
       clipper | \
       cris | \
